Core/Core/Transports/ServerUtils/ServerAPI.cs

```diff
@@ -77,13 +75,9 @@ public async Task<string> DownloadSingleObject(string streamId, string objectId)
       Method = HttpMethod.Get
     };
 
-    HttpResponseMessage rootHttpResponse;
-    do
-    {
-      rootHttpResponse = await _client
-        .SendAsync(rootHttpMessage, HttpCompletionOption.ResponseContentRead, CancellationToken)
-        .ConfigureAwait(false);
-    } while (ShouldRetry(rootHttpResponse));
+    HttpResponseMessage rootHttpResponse = await _client
```

Comment:
we don't call this directly? I think you were saying this is being retried at a higher level, are we 100% sure of this?

Reply:
it's being rertied at a lower level by the http client.
This old code here predates our polly implementation, and we just forgot to remove this nieve do while loop

Reply:
not sure what you mean by "we don't call this directly"